app/assets/javascripts/models/beta/task.js

```diff
@@ -36,6 +36,10 @@ export const destroy = (projectId, storyId, taskId) => {
 export const addTask = (story, task) => (
   {
     ...story,
+    _editing: {
```

Comment:
We should extract this piece of code to some function, something like this:

```js
const setLoadingValue = (story, loading) => ({
  ...story,
  editing
});
```

And then we use it like this:

```js
{
   ...story,
   _editing: setLoadingValue(story._editing, false)
}
```

What do you think? We could do that to other operations like that in a future refactor but this one could be done right now already to remove the duplication of this logic.